### DISCUSSION TITLE
SB 5.1 OCI Provision/DataPipeline/Druid execution fails

### DISCUSSION THREAD

**Discussion by subhashchandrab** [Sunbird-Ed/Community#407] [Installation] [answered]
Trying to follow the documentation https://ed.sunbird.org/use/installing-ed-platform/release-5.1.0/build. The Provision/DataPipeline/Druid fails with following error

MSG:

Failure downloading https://downloads.apache.org/zookeeper/stable/apache-zookeeper-3.6.3-bin.tar.gz, HTTP Error 404: Not Found

The ansible values file from the corresponding release branch https://github.com/Sunbird-Obsrv/sunbird-data-pipeline/blob/release-5.1.0/ansible/roles/zookeeper-upgrade/defaults/main.yml has the following values
zookeeper_version: 3.6.3
zookeeper_url: "https://downloads.apache.org/zookeeper/stable/apache-zookeeper-{{ zookeeper_version }}-bin.tar.gz"

The current stable version for zookeeper is 3.7.4 as per the apache link https://downloads.apache.org/zookeeper/stable/.
Can we go ahead and update the version as 3.7.4 to proceed with installation or do we have any dependencies on the current version being used(3.6.3)

**Comment by Krishnaj20** [comment]
@gandham-santhosh - Pls check and respond to this query.

@chitranshu-keshav @vinukumar-vs @anandp504

**Reply by gandham-santhosh** [reply]
We have another thread for same issue https://github.com/orgs/Sunbird-Ed/discussions/406 
@subhashchandrab Looks like zookeeper removed this artifact recently as it was working when we tried few days back. Please follow above mentioned thread for further discussion.

**Reply by subhashchandrab** [reply]
Sure Santhosh. Will follow the other thread.

**Comment by Krishnaj20** [comment, marked as answer]
We have another thread for same issue https://github.com/orgs/Sunbird-Ed/discussions/406
@subhashchandrab Looks like zookeeper removed this artifact recently as it was working when we tried few days back. Please follow above mentioned thread for further discussion.
